docs/01_nodeos/06_logging/index.md

```diff
@@ -105,4 +106,4 @@ Example:
 ```
 
 [[info]]
-| `net_plugin_impl`, `bnet_plugin`, `producer_plugin`, `transaction_tracing` are not enabled unless explicitly enabled in the `logging.json`
+| All loggers are enabled by default in the `logging.json`. All loggers have the `debug` logging level, except `net_plugin_impl` which is set to `info`.
```

Comment:
The default logging level for all loggers if no `logging.json` is provided is currently `info`. If you provide a `logging.json` file you can set each of loggers to whatever valid value you like.